lib/cli/src/bin/wasmer-headless.rs

```diff
@@ -0,0 +1,61 @@
+use anyhow::Result;
```

Comment:
We don't have to duplicate this; I have a fix locally that doesn't duplicate it

Reply:
Cool, let's address the fix directly on your PR?

Reply:
Fixed